app/deliver_grant_funding/templates/deliver_grant_funding/grant_setup/ggis_number.html

```diff
@@ -0,0 +1,83 @@
+{% extends "common/base.html" %}
+{% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
+{% from "govuk_frontend_jinja/components/error-summary/macro.html" import govukErrorSummary %}
+
+{% block pageTitle %}
+  Do you have a GGIS reference number? - Add a grant - MHCLG Funding Service
+{% endblock pageTitle %}
+
+{% set active_item_identifier = "grants" %}
+
+{% block beforeContent %}
+  {{
+    govukBackLink({
+        "text": "Back",
+        "href": url_for("deliver_grant_funding.grant_setup_intro")
+    })
+  }}
+{% endblock beforeContent %}
+
+{% set before_inputs_text %}
+  <p class="govuk-body govuk-!-margin-top-7">
+    You'll need to provide your GGIS number before you can create forms or assess grant applications.
+  </p>
+{% endset %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <span class="govuk-caption-xl">
+        <span class="govuk-visually-hidden">This section is: </span>Set up the grant
+      </span>
+
+      <form method="post" novalidate>
+        {{ form.csrf_token }}
+
+        {{
+          form.has_ggis(params={
+            "fieldset": {
+              "legend": {
+                "text": form.has_ggis.label.text,
+                "isPageHeading": true,
+                "classes": "govuk-fieldset__legend--xl",
+              },
+            },
+            "formGroup": {
+              "beforeInputs": {
+                "html": before_inputs_text
+              }
+            },
+            "items": [
+              {
+                "value": "yes",
```

Comment:
Can we add a comment to the form field for this, where we declare the choices in a somewhat-duplicative way - letting developers know they need to be kept in sync manually.

This is complex because govuk-frontend-wtf doesn't seem to support conditionals in a nice way.

Hardcoding the values here will affect the browser and what the users sends to the server, but the choices hardcoded in the form will affect validation and what we actually expect. There's a sync issue here.